spec/rspec/mocks/partial_double_spec.rb

```diff
@@ -379,6 +390,22 @@ def defined_private_method
         object.send(:defined_private_method)
       end
 
+      it 'runs the before_verifying_double callbacks on any_instance' do
+        expect { |probe|
+          RSpec.configuration.mock_with(:rspec) do |config|
+            config.before_verifying_doubles(&probe)
+          end
+
+          expect_any_instance_of(klass).to receive(:implemented)
+          object.implemented
+        }.to yield_successive_args(
+          *(klass.ancestors + [klass]).map do |target|
+            have_attributes(:target => target)
+          end
+        )
+      end
+
+
```

Comment:
> Do they verify methods? If they do then we should otherwise it doesn't really matter?

Yes, when `verify_partial_double` is enabled we apply verification to both `expect` and `allow`.  It would be a bug if we did not.  As such I think we should have specs for them here.